update_actions.sh

```diff
@@ -8,23 +8,25 @@ if [ -z "$VERSION" ]; then
         exit 1
 fi
 
-actionsRepos=(delete-namespace
-        build
-        destroy-preview
-        deploy-preview
-        deploy-stack
-        namespace
-        pipeline
-        push
-        create-namespace
-        destroy-pipeline
-        login
-        destroy-stack
-        apply
-        context)
+actionsRepos=(delete-namespace_master
```

Comment:
I think  having repo names with `-` and separating the branch with `_`  could be error prone and probably we would not realize easily. Can we use maybe a tab or another character to separate repo name and branch? I would try to use any char that cannot be "valid" for a repo name

Reply:
yep, we can't use tab because it will detect as another element, but we can use `/`. What do you think?

Reply:
yeah, I don't think we create a repository with `/`